StitchCore/StitchCoreTests/JWTTests.swift

```diff
@@ -0,0 +1,33 @@
+import XCTest
```

Comment:
I'd be much more comfortable importing a library from a third part just for tests.